laboratory/settings.py

```diff
@@ -266,6 +266,8 @@ def SILKY_INTERCEPT_FUNC(request):
 
 QUERY_TIMEOUT = 120
 
+CUSTOM_FORM_100_08_FORMAT = False
```

Comment:
Нужно назвать более понятно, что именно поменяется. Например так:
```suggestion
FORM_100_08_A4_FORMAT = False
```